### ISSUE TITLE
Number widgets overlapping

### ISSUE THREAD

**Issue by JasonEde** [Smashing/smashing#119]
I've just moved my project from dashing to smashing on a new server and in process moved from centos 6 to centos 7. It all works fine apart from my widgets overlapping vertically. I've attached a screen clip to show the issue. It's a sizing issue somewhere, but I can't seem to find where.

In .gridster in application.scss if i change the height: to any value other than inherit it shrinks right down, but too small. I suspect this might be the root cause, but it's not responding to any other value.

![image](https://user-images.githubusercontent.com/946286/49660356-48bc0c80-fa3e-11e8-9dfc-13f6a15340b3.png)

**Comment by TheJefe** [comment]
I solved this using the advice from this comment https://github.com/Shopify/dashing/issues/110#issuecomment-16959053

TLDR;

Add this to application.scss
```
.gridster li {
    overflow: hidden;
    display:  list-item;
}
```

**Comment by kinow** [comment]
@TheJefe if your fix to `application.scss` does not break or alter the dashboard/widgets, and fixes the issue of overlapping widgets, feel free to raise a PR for that 👍

**Comment by TheJefe** [comment]
I'll gladly open a PR, but I don't have enough CSS experience to have a high level of confidence that it won't have some other negative effect.
